pkg/cacheutil/memcached_client.go

```diff
@@ -492,6 +473,27 @@ func (c *memcachedClient) getMultiSingle(ctx context.Context, keys []string) (it
 	return items, err
 }
 
+func (c *memcachedClient) handleError(op string, err error) {
```

Comment:
Maybe "trackError" is more accurate then "handleError", given it just track metrics?